atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/queue/TargetedSweeperTest.java

```diff
@@ -1011,4 +1075,24 @@ private void createAndInitializeSweepersAndWaitForOneBackgroundIteration(int swe
     private void waitUntilBackgroundSweepRunsOneIteration() throws InterruptedException {
         Thread.sleep(3_000);
     }
+
+    private Map<Integer, Integer> enqueueAtLeastThresholdWritesInDefaultShardWithStartTs(long threshold, long startTs) {
+        List<WriteInfo> writeInfos = new ArrayList<>();
+        int counter = 0;
+        while (writeInfos.stream().filter(write -> write.toShard(DEFAULT_SHARDS) == CONS_SHARD).count() < threshold) {
+            writeInfos.addAll(generateHundredWrites(counter++, startTs));
+        }
+        sweepQueue.enqueue(writeInfos);
+        return writeInfos.stream()
+                .collect(Collectors.toMap(write -> write.toShard(DEFAULT_SHARDS), write -> 1, (fst, snd) -> fst + snd));
+    }
```

Comment:
A little hacky, but the point of hashes is to have preimage resistance I guess 😄